### PR TITLE
change to cell_

### PR DESCRIPTION
**Pull request by stemangiola**
I have updated the file 

/vast/projects/cellxgene_curated/metadata_annotated_0.2.2.parquet

to be uploaded.

so the upload and the new release should be possibly done at the same time, Monday would be great, to have time to test in the afternoon before the Tue launch.

Thanks!

### CODE REVIEW

File: R/query.R [hunk 1]
```diff
@@ -177,14 +177,14 @@ group_to_sce <- function(i, df, dir_prefix, features) {
     sce <- loadHDF5SummarizedExperiment(sce_path)
     # The cells we select here are those that are both available in the SCE
     # object, and requested for this particular file
-    cells <- colnames(sce) |> intersect(df$`_cell`)
+    cells <- colnames(sce) |> intersect(df$`cell_`)
```
Comment (multimeric) [inline R/query.R:180]: Remove the backticks from here and below to prove that it is a valid identifier

Comment (stemangiola) [reply R/query.R]: Done

**Comment by multimeric** [comment]
There are a few other changes you need to make. The tests are currently failing, I think because you haven't renamed all instances of `_cell`.

**Comment by stemangiola** [comment]
> There are a few other changes you need to make. The tests are currently failing, I think because you haven't renamed all instances of `_cell`.

apologies for that, done.

**Comment by stemangiola** [comment]
I have updated the metadata version 0.2.3, and the file is here


/vast/projects/cellxgene_curated/metadata_annotated_0.2.3.parquet

**Comment by multimeric** [comment]
Okay it's now available at https://object-store.rc.nectar.org.au/v1/AUTH_06d6e008e3e642da99d806ba3ea629c5/metadata/metadata.0.2.3.parquet. Feel free to update the code to use this.

**Review by multimeric — APPROVED** [review]

**Pull request merged**